transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/repo_grouper.py

```diff
@@ -0,0 +1,150 @@
+import logging
+import os
+from typing import List
+
+import pandas as pd
+import pyarrow as pa
+import ray
+from data_processing.utils import get_logger
+from pyarrow.fs import LocalFileSystem, S3FileSystem
+from pyarrow.parquet import ParquetDataset, write_table
+
+
+class DataAccessAlternative:
```

Comment:
Why do we need data access alternative using S3FS. Thats not good. This is unreliable and uses disk space. Why can't we use data access from the library